x/multisig/model.go

```diff
@@ -13,32 +13,42 @@ const (
 	SequenceName = "id"
 )
 
-// enforce that Contract fulfils desired interface compile-time
+// Weight represents the strength of a signature.
+type Weight uint32
+
+func (w Weight) Validate() error {
+	if w < 1 {
+		return errors.Wrap(errors.ErrInvalidState,
+			"weight must be greater than 0")
+	}
+	return nil
+}
+
 var _ orm.CloneableData = (*Contract)(nil)
 
-// Validate enforces sigs and threshold boundaries
 func (c *Contract) Validate() error {
-	if len(c.Sigs) == 0 {
-		return errors.Wrap(errors.ErrInvalidMsg, "missing sigs")
+	switch n := len(c.Participants); {
+	case n == 0:
+		return errors.Wrap(errors.ErrInvalidMsg, "no participants")
+	case n > 100:
```

Comment:
Nice check here.
Can you pull this into a contant (top of this file), like `const MaxParticipants`?